ui/src/views/AwsProvisioner/ViewAwsWorkerType/index.jsx

```diff
@@ -125,7 +125,7 @@ export default class ViewAwsWorkerType extends Component {
     return (
       <Dashboard title={`AWS Provisioner ${workerType}`}>
         <ErrorPanel fixed error={this.state.error || error} />
-        <Tabs fullWidth value={currentTab} onChange={this.handleTabChange}>
+        <Tabs value={currentTab} onChange={this.handleTabChange}>
```

Comment:
What was the reason for removing `fullWidth`?

Reply:
@helfi92 The deprecated `fullWidth` prop is removed in v4. 

Ref: https://material-ui.com/guides/migration-v3/#tabs